packages/RobotRaconteur/pyproject.toml

```
@@ -0,0 +1,12 @@
+[build-system]
+requires = ["setuptools", "numpy"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "RobotRaconteur"
+version = "1.2.2"
+dependencies = ["numpy"]
+authors = [
+    {name = "John Wason", email = "wason@wasontech.com"}
+]
+license = "Apache-2.0"

```

Ci complains about a missing license field but sure looks like you have a license field here...

---

I am not sure what it is complaining about either. The build for Robot Raconteur is pretty complicated, and the setup.py is generated during the native build so it may be confused. I will keep looking for the cause of the problem.

---

It is coming from the fact that the license is also specified in `setup.py`, and the entry in `pyproject.toml` is overridden by it: https://github.com/robotraconteur/robotraconteur/blob/35c50bbec7a66b5b05d4b585b4c63d536f902fd8/RobotRaconteurPython/setup.py.in#L39

so it would have to be removed from there.

Also, it would be better to store it here as `license = {text = "Apache License (2.0)"}`:

```suggestion
license = {text = "Apache License (2.0)"}
```

---

Could we just remove the license field from `pyproject.toml`?

---

Yes, that works, too